README.md

```diff
@@ -58,19 +58,21 @@ await client.close()
 
 ## API
 
-* [Getting started](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#getting-started)
-* [`attach`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#attach)
-* [`close`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#close)
-* [`connect`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#connect)
-* [`identify`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#identify)
-* [`listPeers`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#listPeers)
-* [`dht.put`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.put)
-* [`dht.get`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.get)
-* [`dht.findPeer`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.findPeer)
-* [`dht.provide`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.provide)
-* [`dht.findProviders`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.findProviders)
-* [`dht.getClosestPeers`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.getClosestPeers)
-* [`dht.getPublicKey`](https://github.com/libp2p/libp2p-daemon-client/blob/master/API.md#dht.getPublicKey)
+* [Getting started](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#getting-started)
+* [`attach`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#attach)
+* [`close`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#close)
+* [`connect`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#connect)
+* [`identify`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#identify)
+* [`listPeers`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#listPeers)
+* [`openStream`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#openStream)
+* [`registerStream`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#registerStream)
+* [`dht.put`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.put)
+* [`dht.get`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.get)
+* [`dht.findPeer`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.findPeer)
+* [`dht.provide`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.provide)
+* [`dht.findProviders`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.findProviders)
+* [`dht.getClosestPeers`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.getClosestPeers)
+* [`dht.getPublicKey`](https://github.com/libp2p/js-libp2p-daemon-client/blob/master/API.md#dht.getPublicKey)
```

Comment:
Making all of these relative urls might be easier. It also makes exploring docs on a branch easier.

```markdown
* [Getting started](./API.md#getting-started)
```